h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamMojoMultinomialModel.java

```diff
@@ -0,0 +1,68 @@
+package hex.genmodel.algos.gam;
+
+import java.util.Arrays;
+
+public class GamMojoMultinomialModel extends GamMojoModelBase {
+    private boolean _multinomial; // true multinomial, false ordinal
+
+    GamMojoMultinomialModel(String[] columns, String[][] domains, String responseColumn) {
+        super(columns, domains, responseColumn);
+    }
+
+    void init() {
+        super.init();
+        _multinomial = _family.equals("multinomial");
+    }
+    
+    @Override
+    double[] gamScore0(double[] row, double[] preds) {
+        if (row.length == nfeatures())
+            _beta_multinomial = _beta_multinomial_center;
+        else
+            _beta_multinomial = _beta_multinomial_no_center;
+        
+        Arrays.fill(preds, 0.0);
```

Comment:
Do we need to fill the preds with zeros ? The preds should be pre-initialized with 0s already. I can see this is copied from GLM. Just that I noticed this.

Reply:
I deleted it.  Thanks.